Bio-KBase-Auth/lib/Bio/KBase/AuthToken.pm

```diff
@@ -585,6 +592,54 @@ sub go_request {
 
 }
 
+sub _get_token {
+    my $self = shift @_;
+    my %p = @_;
+
+    my $json;
+    eval {
+	my $baseurl = $Bio::KBase::Auth::AuthSvcHost;
```

Comment:
In the other clients, the url is passed in, so my feeling is that we should do the same here. The servers should look for auth-server-url in their section of the deploy.cfg as we discussed. 

We might need to alter kb-login to take an auth url temporarily though - not sure. It might be enough just to provide instructions on how to modify the script short term, since the script will stop working once we switch to the new auth server.